README.md

```diff
@@ -83,18 +93,38 @@ See [this author page](https://github.com/ait-energy/en.ergie.at/blob/main/conte
 
 ## Development
 
-### Local setup
+### Local setup (Linux)
 
 1. Install Hugo: [gohugo.io/installation](https://gohugo.io/installation/)
 2. Clone this repository, e.g., `git clone https://github.com/ait-energy/en.ergie.at.git`
 3. Inside the folder `en.ergie.at`: execute `git submodule update --init --recursive` to also get the (submodule-d) theme (this might take a bit)
 4. Inside the folder `en.ergie.at`: execute `git submodule update --remote --merge` to update the theme
 5. Inside the folder `en.ergie.at`: execute `hugo server --buildDrafts`
 
+### Local Setup (Windows)
+
+Cloning a repository
+
+1. Open the repository's entry page in your browser  
+2. Click on "Code" → Copy the provided URL  
+3. Open a new terminal in VSCode
+4. Navigate to your development folder using: `cd path/to/dev_folder`  
+5. Clone the repository by entering: `git clone URL` (paste the repository's URL)
+
+Setting up the local environment
+
+1. Install Hugo: gohugo.io/installation
+2. Extract the `hugo.exe` file and copy it into your development folder  
+3. Open a new terminal in VSCode
+4. Inside the folder `en.ergie.at`, execute `git submodule update --init --recursive` to also get the (submodule-d) theme (this might take a bit)  
+5. Inside the folder `en.ergie.at`, execute `git submodule update --remote --merge` to update the theme  
+6. Inside the folder `en.ergie.at`, execute `./hugo.exe server --buildDrafts` to start the local server  
```

Comment:
```suggestion
6. Inside the folder `en.ergie.at`, execute `./hugo server --buildDrafts` to start the local server  
```

Reply:
Not 100% sure if that works, please check. But if it does, it's the same as in the Linux guide and then we can later merge that somehow.

Reply:
I'm pretty sure it works. We first tried the linux version (without ./) and it didnt work, I don't remember how you then figured out that we needed to write ./hugo.exe but I took a screenshot with it working